python/veles/async_client/conn.py

```diff
@@ -22,7 +20,6 @@
     QueryHandler,
     BroadcastHandler,
 )
-from veles.async_conn.node import AsyncNode
 from veles.async_conn.subscriber import (
```

Comment:
pylint:
>E: 23, 0: No name 'BaseSubscriberNode' in module 'veles.async_conn.subscriber' (no-name-in-module)
E: 23, 0: No name 'BaseSubscriberData' in module 'veles.async_conn.subscriber' (no-name-in-module)
E: 23, 0: No name 'BaseSubscriberBinData' in module 'veles.async_conn.subscriber' (no-name-in-module)
E: 23, 0: No name 'BaseSubscriberList' in module 'veles.async_conn.subscriber' (no-name-in-module)